### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by pereirinha**
Updates the readme information for the plugin

## Approach

- n/a

## QA notes

- You can't see these updates before launching them, but you can use the following to validate and preview them:
  - https://wordpress.org/plugins/developers/readme-validator/
  - https://wpreadme.com/

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -8,37 +8,35 @@ Stable tag: STABLETAG
...
+Cloudinary’s [award-winning plugin](https://www.businesswire.com/news/home/20200923005566/en/Cloudinary-Wins-2020-MarTech-Breakthrough-Award-for-Best-WordPress-Plugin) makes automating your image and video workflows easy, offering a variety of features. Supporting nearly two million developers and over 10,000 customers, Cloudinary enables companies to manage, transform, optimize and deliver image and video assets. The Cloudinary Plugin supports all these capabilities, providing users with a true plug-and-play solution. Getting started takes only minutes using the intuitive onboarding wizard.
 
 https://youtu.be/AMVS_O_VEss
 
 > Know more on at the [Cloudinary Academy - Introduction to Cloudinary for WordPress Administrators](https://training.cloudinary.com/courses/introduction-to-cloudinary-for-wordpress-administrators-70-minute-course-zf3x)
```
Comment (yuval-cloudinary) [inline readme.txt:31]: Maybe replace "Know more on at the [Cloudinary Academy..." with "Learn more about the plugin with the [Cloudinary Academy..."

File: readme.txt [hunk 2]
```diff
@@ -8,37 +8,35 @@ Stable tag: STABLETAG
...
+Cloudinary’s [award-winning plugin](https://www.businesswire.com/news/home/20200923005566/en/Cloudinary-Wins-2020-MarTech-Breakthrough-Award-for-Best-WordPress-Plugin) makes automating your image and video workflows easy, offering a variety of features. Supporting nearly two million developers and over 10,000 customers, Cloudinary enables companies to manage, transform, optimize and deliver image and video assets. The Cloudinary Plugin supports all these capabilities, providing users with a true plug-and-play solution. Getting started takes only minutes using the intuitive onboarding wizard.
 
 https://youtu.be/AMVS_O_VEss
```
Comment (yuval-cloudinary) [inline readme.txt:29]: I'm not sure what was expected to be here, it takes me to a promotional of Atera.com

File: readme.txt [hunk 3]
```diff
@@ -8,37 +8,35 @@ Stable tag: STABLETAG
...
 
 > Know more on at the [Cloudinary Academy - Introduction to Cloudinary for WordPress Administrators](https://training.cloudinary.com/courses/introduction-to-cloudinary-for-wordpress-administrators-70-minute-course-zf3x)
 
-On top of that, users can also leverage a dashboard giving insights into optimization metrics and data usage so there are never any surprises. No matter if your use case is for personal use or business needs, the Cloudinary plugin offers enterprise quality and architecture for all.
+After deployment, users can access a dashboard that provides insights into  optimization metrics and data usage. The Cloudinary Plugin is based on an architecture that supports any type of site including business, enterprise and personal.
```
Comment (yuval-cloudinary) [inline readme.txt:33]: "into  optimization" has a double space

File: readme.txt [hunk 4]
```diff
@@ -8,37 +8,35 @@ Stable tag: STABLETAG
...
+**Optimize your site performance** and user experience by achieving faster page load times and implementing advanced auto-responsive delivery. This not only improves your **Google page rankings** but also enhances **Core Web Vitals**, and **Lighthouse scores**. By leveraging features such as video and image optimization, advanced responsive design, customizable lazy loading, and built-in CDN support, you can publish content faster while consistently delivering engaging user experiences. The Plugin supports both headless and UI-based development models,  and allows you to customize development and extend functionality with the Cloudinary Plugin [actions and filters](https://cloudinary.com/documentation/wordpress_developers#actions_and_filters). Moreover, you can create engaging visual experiences with automated quality and encoding settings, **video and image resizing** and **cropping via AI**, comprehensive [transformations](https://cloudinary.com/documentation/image_transformations) and effects, and seamless delivery to any device in any resolution or pixel density.
```
Comment (yuval-cloudinary) [inline readme.txt:35]: "models,  and" has a double space

**Review by yuval-cloudinary — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 2 above)
Comment (yuval-cloudinary) [reply readme.txt:29]: > I'm not sure what was expected to be here, it takes me to a promotional of Atera.com

that was an ad, and irrelevant for embedded in page

File: readme.txt (hunk 3 above)
Comment (pereirinha) [reply readme.txt]: Good catch

**Review by yuval-cloudinary — APPROVED** [review]

**Pull request merged**